packages/client/components/auth/src/flows/Form.tsx

```diff
@@ -72,6 +80,8 @@ export function Fields(props: FieldProps) {
   const fieldConfiguration = useFieldConfiguration();
   const [failedValidation, setFailedValidation] = createSignal(false);
 
+  const inviteCodeNeeded: boolean|undefined = clientController.lifecycle.client.configuration?.features.invite_only;
```

Comment:
A simple missed formatting:
```suggestion
  const inviteCodeNeeded: boolean | undefined = clientController.lifecycle.client.configuration?.features.invite_only;
```